### DISCUSSION TITLE
Open TV is coming to mobile. Consider donating to help fund this project.

### DISCUSSION THREAD

**Discussion by Fredolx** [Fredolx/open-tv#66] [Announcements]
Open TV is coming to Android, Android TV and iOS.
Please consider donating to support the creation of the first ever open-source mobile IPTV app.

**Comment by Ahm074** [comment]
Hi, super powerful fast...what else.?

Thanks for the work, I'm impressed with your app. The graphical interface is successful, and no freezes since I've been using it.
I'm really looking forward to testing the mobile version.
Thanks again

**Comment by Fredolx** [comment]
Some progress

<img width="614" alt="Screenshot 2025-01-26 at 12 58 39 AM" src="https://github.com/user-attachments/assets/abad5b63-4710-43b5-a959-5a855e875e2d" />

**Comment by Fredolx** [comment]
Showcase of error reporting & adding new source


https://github.com/user-attachments/assets/d9cf6d82-ded0-4edc-8a3e-459eadfa2a3e

**Comment by Fredolx** [comment]
https://github.com/user-attachments/assets/eba06eaa-fda7-48d1-9c4f-8c6d2fc66c08

**Comment by Fredolx** [comment]
https://github.com/user-attachments/assets/018e42d3-844c-4d1d-ae1e-0f518cc86bd1

**Comment by Fredolx** [comment]
https://github.com/user-attachments/assets/a62e0d67-dc2d-4eeb-9f2e-aea39b0f1846
